### PR TITLE
Cleanup various deprecation warnings.

### PR DESCRIPTION
**Pull request by abutcher**
Working towards v2 support https://github.com/openshift/openshift-ansible/issues/686

### CODE REVIEW

**Comment by abutcher** [comment]
aos-ci-test

File: playbooks/common/openshift-etcd/config.yml [hunk 1]
```diff
@@ -53,15 +53,15 @@
         -C {{ etcd_generated_certs_dir }}/{{ item.etcd_cert_subdir }} .
     args:
       creates: "{{ etcd_generated_certs_dir }}/{{ item.etcd_cert_subdir }}.tgz"
-    with_items: etcd_needing_server_certs
+    with_items: "{{ etcd_needing_server_certs }}"
```
Comment (detiber) [inline playbooks/common/openshift-etcd/config.yml]: Ah, so ansible 2.0 prefers explicit templates here instead of the implicit ones, nice!

Comment (abutcher) [inline playbooks/common/openshift-etcd/config.yml]: Yeah! Still testing 1.9.4 with this -- looks like a lot of these need defaults when templated.

Comment (detiber) [inline playbooks/common/openshift-etcd/config.yml]: I wonder if this will clean up some of the hard to reproduce issues we've seen then...

**Comment by abutcher** [comment]
aos-ci-test

**Comment by detiber** [comment]
@abutcher are you still testing this?

**Comment by abutcher** [comment]
@detiber Yep, confirming subsequent runs like the defaults.

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
@detiber good to go pending tests

**Comment by detiber** [comment]
@abutcher suspicious failure...

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

File: roles/openshift_master_facts/tasks/main.yml [hunk 2]
```diff
@@ -22,7 +22,6 @@
       extension_scripts: "{{ openshift_master_extension_scripts | default(None) }}"
       extension_stylesheets: "{{ openshift_master_extension_stylesheets | default(None) }}"
       extensions: "{{ openshift_master_extensions | default(None) }}"
-      oauth_template: "{{ openshift_master_oauth_template | default(None) }}"
```
Comment (detiber) [inline roles/openshift_master_facts/tasks/main.yml]: Did you mean to remove oauth_template here?

Comment (abutcher) [inline roles/openshift_master_facts/tasks/main.yml]: Yessir

**Pull request merged**